NightscoutServiceKit/RemoteCommands/Validators/RemoteCommandValidator.swift

```diff
@@ -6,6 +6,57 @@
 //  Copyright © 2023 LoopKit Authors. All rights reserved.
 //
 
```

Comment:
Part of the validation refactor. I moved the little validator classes in this one class. It avoids the awkwardness of passing the `otpManager` into the previous function calls like `remoteNotification.validate(otpManager)`